main.scss

```diff
@@ -1,38 +1,48 @@
 /*
 
-This includes all the basic origami styles and mixins you can
+This includes all the basic Origami styles and mixins you can
 expect to be available when a page uses n-ui
 
 */
 
 // Base unit to standardize spacing
 // Decide where it's gonna live later
-$spacing-unit: 16px;
+$spacing-unit: 20px;
```

Comment:
this used much?

Reply:
only [here and there](https://github.com/search?q=org%3AFinancial-Times+spacing-unit&type=Code), but the plan was to align it with what the design folks want (& use), and leave any decisions about its adoption/usefulness for another time. Seemed like a bit of a rabbit hole